curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which CSS property is used to specify the image to be used as a list item marker?
 
 #### --distractors--
 
-Placeholder distractor 1
+list-style-position
```

Comment:
```suggestion
`list-style-position`
```

you need to use backticks around anything code, even when they are the only word in an answer/distractor

Reply:
Oh my bad, I have added that for code which are in questions but didn't do the same for options. Anyhow will commit the suggested changes, please give some time. Thank you!

Reply:
Which of the following is NOT a valid value for the `list-style-type` property?

#### --distractors--

decimal

---

lower-greek

---

square

#### --answer--

lower-arabic


@ilenia-magoni I just wanted to know if i should use ` ` for values as well. As you can see the above example in that options are values so should I put them as well inside the ` ` ?

Reply:
exactly those needs to go in backticks